kiosk/src/Components/ErrorModal.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import { tickEvent } from "../browserUtils";
+import "../Kiosk.css";
+
+
+interface IProps {
+    errorType: string;
+    errorDescription: string;
+    setShowing: (p: string) => void;
+}
+const ErrorModal: React.FC<IProps> = ({ errorType, errorDescription, setShowing }) => {
+    const cancelClicked = () => {
+        const modal = document.getElementsByClassName("common-modal-container")[0];
+        while (modal.firstChild) {
```

Comment:
This DOM manipulation is unusual. Why is it needed?

Reply:
Oh. This is a lapse on my part. When I was thinking about how to get modals to close, I think I just followed the pattern that we use for injecting and then getting rid of the iframe for game play. The iframe stuff is done purely with JavaScript and so I just followed what made sense to me. It just completely slipped my mind that I should just be using the variables that I already have for the conditional rendering. Thank you for pointing this out. 

I have this logic in the modal for deletion, as well. I will get rid of it there, too.